src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/utilities/translations.ts

```diff
@@ -427,6 +427,9 @@ export let TRANSLATIONS = {
     DownloadError: {
       en: 'There was a problem with the download',
     },
+    TooManyContentsError: {
+      en: 'The download of more than 1000 contents is not supported.',
```

Comment:
Debatable but, in my opinion, **contents** doesn't fit in the phrase. For example, **items** would be more suitable

Reply:
agreed